kitsune/search/v2/signals/__init__.py

```diff
@@ -0,0 +1 @@
+from . import questions, users, wiki, forums  # noqa
```

Comment:
why is this needed?

Reply:
So `kitsune/search/apps.py:SearchV2Config` can set up all the signals at once with just `from kitsune.search.v2 import signals`